python/ql/src/semmle/python/objects/ObjectAPI.qll

```diff
@@ -573,6 +573,9 @@ class ClassValue extends Value {
 abstract class FunctionValue extends CallableValue {
     abstract string getQualifiedName();
 
+    /** Gets a longer, more descriptive version of toString() */
+    abstract string descriptiveString();
```

Comment:
I'm curious as to the reason for adding this, since it's not used anywhere in this PR. Why not change `toString` to be more descriptive?

Reply:
This is something that was also added in other PRs, and the method is used by the original query. We can improve `toString` but that will need to be a separate project, since `descriptiveString` is used in a bunch of queries, and also since changing `toString` will mean changing a LOT of query results.

Reply:
The method wouldn't need to be added here but for the fact that other PRs haven't merged yet, and this wouldn't typecheck without `descriptiveString`.

Reply:
Ah, I get it. I had not seen that `descriptiveString` was actually used on line 37 of `IncorrectlySpecifiedOverriddenMethod.ql` (because there was no diff for that part, and so it was hidden from my view). :man_facepalming: My apologies.